# Igor Gorbounov <igor.gorbounov@gmail.com>, 2019. #zanata, 2020, 2021.
# Vitaliy Bukatkin <vitaliy.bukatkin@gmail.com>, 2020.
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-06-06 09:18-0400\n"
"PO-Revision-Date: 2021-02-11 20:40+0000\n"
"Last-Translator: Igor Gorbounov <igor.gorbounov@gmail.com>\n"
"Language-Team: Russian <https://translate.fedoraproject.org/projects/dnf/dnf-"
"plugins-extras/ru/>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Generator: Weblate 4.4.2\n"

#: plugins/kickstart.py:57
msgid "Install packages defined in a kickstart file on your system"
msgstr "Установка пакетов, определенных в файле кикстарта, в систему "

#: plugins/kickstart.py:62
msgid "kickstart file"
msgstr "файл кикстарта"

#: plugins/kickstart.py:80
#, python-format
msgid "file cannot be parsed: %s"
msgstr "не удается разобрать файл: %s"

#: plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "Добавление «{}» в список обработки пакетов для rpmconf"

#: plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr "модуль rpmconf не будет работать в неинтерактивном режиме"

#: plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr "пропуск sys.exit из rpmconf из-за отсутствия переменной MERGE"

#: plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr "пропуск sys.exit из rpmconf из-за отсутствия файла"

#: plugins/showvars.py:46
msgid "show all active dnf variables"
msgstr "показать все активные переменные dnf"

#: plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "не удалось подключиться к snapperd: %s"

#: plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr "создание предварительного снимка"

#: plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr "создан предварительный снимок %d"

#: plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr "не удалось создать предварительный снимок: %s"

#: plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""
"пропускается последующий снимок, потому что не удалось создать "
"предварительный снимок"

#: plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr "создание последующего снимка"

#: plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr "создан последующий снимок %d"

#: plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr "не удалось создать последующий снимок: %s"

#. Translators: This string is only used in unit tests.
#: plugins/system_upgrade.py:45
msgid "the color of the sky"
msgstr "цвет неба"

#: plugins/system_upgrade.py:57
msgid "Need a --releasever greater than the current system version."
msgstr "--releasever должно быть больше текущей версии системы."

#. Translators: do not change "reboot" here
#: plugins/system_upgrade.py:59
#, python-brace-format
msgid ""
"Download complete! Use 'dnf {command} reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf {command} clean'"
msgstr ""
"Загрузка завершилась! Для запуска обновления используйте «dnf {command} "
"reboot».\n"
"Для удаления кэшированных метаданных и транзакции используйте «dnf {command} "
"clean»"

#: plugins/system_upgrade.py:62
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""
"К сожалению, необходимо использовать «download --releasever» вместо «--"
"network»"

#: plugins/system_upgrade.py:71
msgid "Reboot turned off, not rebooting."
msgstr "Перезагрузка отключена, перезагрузка не выполняется."

#: plugins/system_upgrade.py:120
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr "Гашение экрана не отключается: %s"

#: plugins/system_upgrade.py:140
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr ""
"Не удалось загрузить файл состояния: %s, продолжение с пустым состоянием."

#: plugins/system_upgrade.py:286
msgid "The following boots appear to contain upgrade logs:"
msgstr "Похоже, что в следующих загрузках есть журналы обновления:"

#: plugins/system_upgrade.py:296
msgid "-- no logs were found --"
msgstr "-- журналы не найдены --"

#: plugins/system_upgrade.py:311
msgid "Cannot find logs with this index."
msgstr "Не удается найти журналы с этим индексом."

#: plugins/system_upgrade.py:320
msgid "Unable to match systemd journal entry"
msgstr "Не удалось найти соответствие записи в журнале systemd"

#: plugins/system_upgrade.py:341
msgid "Prepare system for upgrade to a new release"
msgstr "Подготовка системы к обновлению до новой версии"

#: plugins/system_upgrade.py:357
msgid "keep installed packages if the new release's version is older"
msgstr "оставить установленные пакеты, если версия нового выпуска старше"

#: plugins/system_upgrade.py:361
msgid "which logs to show"
msgstr "какие журналы показать"

#: plugins/system_upgrade.py:395
#, python-brace-format
msgid "Incompatible version of data. Rerun 'dnf {command} download [OPTIONS]'"
msgstr ""
"Несовместимая версия данных. Еще раз запустите «dnf {command} download "
"[ПАРАМЕТРЫ]»"

#: plugins/system_upgrade.py:432
msgid "Command 'offline-distrosync' cannot be used with --no-downgrade option"
msgstr ""
"Команду «offline-distrosync» нельзя использовать с параметром --no-downgrade"

#: plugins/system_upgrade.py:454
msgid "Additional information for System Upgrade: {}"
msgstr "Дополнительная информация для обновления системы: {}"

#: plugins/system_upgrade.py:457
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""
"Прежде, чем продолжить, убедитесь, что ваша система полностью обновлена, с "
"помощью команды «dnf --refresh upgrade». Хотите продолжить"

#: plugins/system_upgrade.py:461
msgid "Operation aborted."
msgstr "Операция отменена."

#: plugins/system_upgrade.py:519
msgid "system is not ready for upgrade"
msgstr "система не готова к обновлению"

#: plugins/system_upgrade.py:522
#, python-brace-format
msgid ""
"the transaction was not prepared for '{command}'. Rerun 'dnf {command} "
"download [OPTIONS]'"
msgstr ""
"транзакция не была подготовлена для «{command}». Еще раз запустите «dnf "
"{command} download [ПАРАМЕТРЫ]»"

#: plugins/system_upgrade.py:526
msgid "upgrade is already scheduled"
msgstr "обновление уже запланировано"

#: plugins/system_upgrade.py:532
msgid "trigger file does not exist. exiting quietly."
msgstr "триггерный файл не существует, завершаем работу."

#: plugins/system_upgrade.py:535
msgid "another upgrade tool is running. exiting quietly."
msgstr "работает еще одно средство обновления, завершаем работу."

#: plugins/system_upgrade.py:544
#, python-brace-format
msgid "use 'dnf {command} reboot' to begin the upgrade"
msgstr "используйте «dnf {command} reboot», чтобы начать обновление"

#: plugins/system_upgrade.py:562
msgid "Rebooting to perform upgrade."
msgstr "Перезагрузка для выполнения обновления."

#: plugins/system_upgrade.py:595
msgid "Starting offline upgrade. This will take a while."
msgstr "Начинается автономное обновление. На это понадобится некоторое время."

#: plugins/system_upgrade.py:597
msgid "Starting offline distrosync. This will take a while."
msgstr ""
"Начинается автономная синхронизация дистрибутива. На это понадобится "
"некоторое время."

#: plugins/system_upgrade.py:599
msgid "Starting system upgrade. This will take a while."
msgstr "Запуск обновления системы. На это потребуется какое-то время."

#: plugins/system_upgrade.py:615
msgid "Cleaning up downloaded data..."
msgstr "Очистка загруженных данных..."

#: plugins/system_upgrade.py:644
msgid ""
"The system-upgrade transaction is empty, your system is already up-to-date."
msgstr ""

#: plugins/system_upgrade.py:653
msgid "Transaction saved to {}."
msgstr "Транзакция сохранена в {}."

#: plugins/system_upgrade.py:656
msgid "Error storing transaction: {}"
msgstr "Ошибка при сохранении транзакции: {}"

#: plugins/system_upgrade.py:678
msgid "Download finished."
msgstr "Загрузка закончена."

#: plugins/system_upgrade.py:681 plugins/system_upgrade.py:682
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "Обновление завершилось! Очистка и перезагрузка..."

#: plugins/system_upgrade.py:691
msgid "Prepare offline upgrade of the system"
msgstr "Подготовить автономное обновление системы"

#: plugins/system_upgrade.py:696
msgid "Prepare offline distrosync of the system"
msgstr "Подготовить автономную синхронизацию дистрибутива системы"

#: plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr "Отключение модуля torproxy: не удается подключиться к сети Tor"

#: plugins/tracer.py:69
msgid "For more information run:"
msgstr "Для подробной информации запустите:"

#: plugins/tracer.py:96
msgid "Call to Tracer API ended unexpectedly:"
msgstr "Вызов Tracer API завершился неожиданно:"

#: plugins/tracer.py:98
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""
"Посетите https://github.com/FrostyX/tracer/issues и отправьте сообщение о "
"проблеме. Спасибо"

#: plugins/tracer.py:100
msgid "We apologize for any inconvenience"
msgstr "Приносим извинения за неудобства"

#~ msgid "Unable to match package: %s"
#~ msgstr "Не удалось найти соответствие пакету: %s"

#~ msgid "Unable to match some of packages"
#~ msgstr "Не удалось найти соответствие некоторым пакетам"
